server/service/src/auth.rs

```diff
@@ -244,6 +245,11 @@ fn all_permissions() -> HashMap<Resource, PermissionDSL> {
         Resource::QueryInventoryAdjustmentReasons,
         PermissionDSL::NoPermissionRequired,
     );
+    map.insert(
+        Resource::QueryStorePreferences,
+        PermissionDSL::NoPermissionRequired,
```

Comment:
Don't you need store permisisons? `PermissionDSL::HasStoreAccess`

Reply:
[716b173](https://github.com/openmsupply/open-msupply/pull/1250/commits/716b173bf1fffdcd625abc7f7724ffa28c016f10)

Reply:
[ff19359](https://github.com/openmsupply/open-msupply/pull/1250/commits/ff193598e53cbe5e271a52069ee47c0703a463ba)